authserver/db/models/models.py

```diff
@@ -133,7 +133,7 @@ class Meta:
     lastname = fields.String(required=True)
     organization = fields.String(required=True)
     email_address = fields.Email(required=True)
-    telephone = fields.String(required=True)
+    telephone = fields.String(required=False)
```

Comment:
@gregmundy - Let's just omit `required=False`, since that would be the default setting (and the conventional way of handing this). I also find the code easier to read without the explicit setting.